docs/how-to/integrate-platform.md

```diff
@@ -48,18 +48,20 @@ To connect your PMM server to Percona Platform:
 To confirm that you have successfully connected the server and check the list of all servers currently connected to an organization, go to [Percona Platform](https://portal.percona.com) > **Dashboard** tab and click **View Instances** next to the **Connect your PMM** step. 
 
 ## Check Percona Portal entitlements
-After connecting to the Percona Platform, PMM has access to additional alert templates,   Advisors checks, and account information. See (../how-to/account-info.md)
-
-### Disconnect a PMM instance
- Disconnect a PMM instance when you want to unlink it from your Percona Platform organization or stop monitoring it there. 
-
-To disconnect a PMM server, go to > <i class="uil uil-cog"></i> **Configuration > Settings > Percona Platform** and click **Disconnect**. 
-
-To confirm that the server disconnected successfully, go to Percona Platform >  **Dashboard** tab > **View instances**. This displays the **PMM instances** page where you can check the list of servers currently connected to your Platform organization. 
+After connecting to the Percona Platform, PMM has access to additional alert templates, Advisors checks, and account information. See (../how-to/account-info.md)
 
 ## Sign into PMM with your Percona Account
 Once you've successfully connected your PMM instance to the Percona Platform, you can also sign into PMM using your Percona Account:
 
 1. Log out of your existing PMM session.
-2. On the PMM login screen, click *Sign in with Percona Account*. 
- If you have an active Percona Account session on the same browser, PMM will log you in automatically. Otherwise, enter your Percona Account credentials to start a new session.
\ No newline at end of file
+2. On the PMM login screen, click **Sign in with Percona Account**. 
+ If you have an active Percona Account session on the same browser, PMM will log you in automatically. Otherwise, enter your Percona Account credentials to start a new session.
+
+### Disconnect a PMM instance
+Disconnect a PMM instance when you want to unlink it from your Percona Platform organization and stop monitoring it there. PMM Admins can also force-disconnect servers when troubleshooting problematic connections.
```

Comment:
this would change the meaning. This is a shortdesc that explain when it would be necessary to disconnect, so that they decide if they want to do this. 
Also, we don't use "the" in front of the product name